homeassistant/components/sun/__init__.py

```diff
@@ -170,6 +170,7 @@ def update_events(self, utc_point_in_time):
             utc_point_in_time, 'dusk', PHASE_ASTRONOMICAL_TWILIGHT)
         self.next_midnight = self._check_event(
             utc_point_in_time, 'solar_midnight', None)
+        self.location.solar_depression = 'civil'
```

Comment:
Since you call it a reset, why not store the original value?

Reply:
Going back to the default value after fiddling with it. 

Location probably shouldn't really store state like that either, but it does and it's not our library.

It might better to have it as a parameter to our wrapper function and default it to civil anyhow, but I don't really want to mess with it since this break happened.